### PR TITLE
Snake cog

### PR DESCRIPTION
**Pull request by lemonsaurus**
The API's needed for the snake cog.

### CODE REVIEW

File: pysite/views/api/bot/hiphopify.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 class HiphopifyView(APIView, DBMixin):
     path = "/hiphopify"
-    name = "hiphopify"
+    name = "api.bot.hiphopify"
```
Comment (gdude2002) [inline pysite/views/api/bot/hiphopify.py]: Nope, just `bot.hiphopify` - although we aren't namespacing this correctly anyway, it should really be `/bot/hiphopify` if we're doing that

File: pysite/views/api/bot/snake_cog/snake_facts.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from pysite.base_route import APIView
+from pysite.decorators import api_key
+from pysite.mixins import DBMixin
+
+log = logging.getLogger(__name__)
+
+
+class SnakeFactsView(APIView, DBMixin):
+    path = "/snake_facts"
+    name = "api.bot.snake_facts"
```
Comment (gdude2002) [inline pysite/views/api/bot/snake_cog/snake_facts.py]: See hiphopify comment

File: pysite/views/api/bot/snake_cog/snake_idioms.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+from pysite.base_route import APIView
+from pysite.decorators import api_key
+from pysite.mixins import DBMixin
+
+log = logging.getLogger(__name__)
+
+
+class SnakeIdiomView(APIView, DBMixin):
+    path = "/snake_idioms"
+    name = "api.bot.snake_idioms"
```
Comment (gdude2002) [inline pysite/views/api/bot/snake_cog/snake_idioms.py]: See hiphopify comment

File: pysite/views/api/bot/snake_cog/snake_names.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+GET_SCHEMA = Schema([
+    {
+        Optional("get_all"): str
+    }
+])
+
+
+class SnakeNamesView(APIView, DBMixin):
+    path = "/snake_names"
+    name = "api.bot.snake_names"
```
Comment (gdude2002) [inline pysite/views/api/bot/snake_cog/snake_names.py]: See hiphopify comment

File: pysite/views/api/bot/snake_cog/snake_quiz.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+from pysite.base_route import APIView
+from pysite.decorators import api_key
+from pysite.mixins import DBMixin
+
+log = logging.getLogger(__name__)
+
+
+class SnakeQuizView(APIView, DBMixin):
+    path = "/snake_quiz"
+    name = "api.bot.snake_quiz"
```
Comment (gdude2002) [inline pysite/views/api/bot/snake_cog/snake_quiz.py]: See hiphopify comment

File: pysite/views/api/bot/snake_cog/special_snakes.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+from pysite.base_route import APIView
+from pysite.decorators import api_key
+from pysite.mixins import DBMixin
+
+log = logging.getLogger(__name__)
+
+
+class SpecialSnakesView(APIView, DBMixin):
+    path = "/special_snakes"
+    name = "api.bot.special_snakes"
```
Comment (gdude2002) [inline pysite/views/api/bot/snake_cog/special_snakes.py]: See hiphopify comment

**Review by gdude2002 — CHANGES_REQUESTED** [review]

File: pysite/migrations/tables/snake_facts/initial_data.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
+[
+   {"fact": "The decapitated head of a dead snake can still bite, even hours after death. These types of bites usually contain huge amounts of venom."},
```
Comment (jerbob) [inline pysite/migrations/tables/snake_facts/initial_data.json]: Can we have 4 spaces of indentation in this file, to remain consistent with `initial_data.json`?

File: pysite/migrations/tables/snake_quiz/initial_data.json [hunk 8]
```diff
@@ -0,0 +1,200 @@
+[
+  {
```
Comment (jerbob) [inline pysite/migrations/tables/snake_quiz/initial_data.json]: iNcOnSIStent JsON iNDeNtaTion

**Review by jerbob — CHANGES_REQUESTED** [review]

**Review by gdude2002 — DISMISSED** [review]

**Review by jerbob — APPROVED** [review]

**Comment by jerbob** [comment]
danke mein freund

**Pull request merged**